src/main/java/net/minestom/arena/game/Arena.java

```diff
@@ -9,4 +9,6 @@ public interface Arena {
     @NotNull Group group();
 
     @NotNull CompletableFuture<Void> init();
+
+    int stage();
```

Comment:
Is it necessary? Not all arena may be wave/stage-based

Reply:
Fixed in latest commit